lib/XUnion.js

```diff
@@ -20,7 +20,7 @@ class XUnion {
     try {
       this.db = new DB(this.config.db);
       this.orderBook = new OrderBook(this.db);
-      this.lndClient = new LndClient(this.config.lndDir);
+      this.lndClient = new LndClient(this.config.lndDir, this.config.lndRpcProto);
```

Comment:
Maybe just pass in the `config` object in its entirety and then read the properties we need in the LndClient constructor?

Reply:
Definitely.